zkevm-circuits/src/copy_circuit.rs

```diff
@@ -680,7 +680,6 @@ impl<F: Field> CopyCircuit<F> {
     }
 }
 
-#[cfg(feature = "dev")]
```

Comment:
I think this should be:

```
#[cfg(any(feature = "test", test))]
```

Reply:
Added `#[cfg(any(feature = "test", test))]` to mod `dev` in `copy_circuit`.